testutils/testutils.go

```diff
@@ -1,8 +1,11 @@
 package testutils
 
 import (
+	"github.com/kyma-project/nats-manager/api/v1alpha1"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
 func NewTestLogger() (*zap.Logger, error) {
```

Comment:
This is inside the `testutils` package, the word `test` sounds redundant:

`NewTestLogger` -> `NewLogger`

Reply:
updated